x-pack/plugins/observability_solution/profiling/public/components/topn_functions/index.tsx

```diff
@@ -316,7 +318,7 @@ export const TopNFunctionsGrid = ({
           showColumnSelector: false,
           showKeyboardShortcuts: !isDifferentialView,
           showDisplaySelector: !isDifferentialView,
-          showFullScreenSelector: !isDifferentialView,
+          showFullScreenSelector: showFullScreenSelector ?? !isDifferentialView,
```

Comment:
We're defining a default value for `showFullScreenSelector`, so `!iDifferentialView` will never be checked. Should it be: `showFullScreenSelector && !isDifferentialView`?

Reply:
The idea was to be undefined by default so we can first check that if we have it and only if it's not set to check `!isDifferentialView` - then I added the default, but you are right I will change it to be true by default 👍